app/javascript/components/ConversationPage.js

```diff
@@ -52,40 +47,17 @@ class ConversationPage extends Component {
       return (
         <Message
           key={ `${index} + ${sent_on}` }
-          sender_first_name={ sender_first_name }
+          sender_first_name={ [sender_first_name] }
```

Comment:
I'm getting an error about the Message component expecting a String variable for sender_first_name rather than an array whilst viewing the inbox